### ISSUE TITLE
Misleading error message when attempting to generate a custom check with the name of an existing file

### ISSUE THREAD

**Issue by michaelkpfeifer** [rrrene/credo#936]
### Precheck

* Proposals for new features should be submitted via: https://github.com/rrrene/credo-proposals
* For bugs, please do a quick search and make sure the bug has not yet been reported here

### Environment

* Credo version (`mix credo -v`): 1.6.1
* Erlang/Elixir version (`elixir -v`): 1.13.0
* Operating system: Debian GNU/Linux 10

### What were you trying to do?

Attempt to generate a custom check with the name of an existing file
```
$ mix credo gen.check lib/test01.ex # first attempt works as expected, creates li/test01.ex
$ mix credo gen.check lib/test01.ex # second attempt prints misleading error message
```

### Expected outcome

Expected error message
```
File exists: lib/test01.ex, aborted.
```

### Actual outcome

Actual error message
```
Please provide a filename:

  mix credo gen.check lib/my_first_credo_check.ex

```

**Comment by rrrene** [comment]
Thx for reporting! :+1:

**Comment by rrrene** [comment]
Thanks for reporting this 😀  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍

**Comment by rrrene** [comment]
This is part of `v1.6.4`. Thx for reporting!
